templates/commands/render/action.go

```diff
@@ -47,26 +47,26 @@ type walkAndModifyVisitor func([]byte) ([]byte, error)
 //
 // If the visitor returns modified file contents for a given file, that file
 // will be overwritten with the new contents.
-func walkAndModify(ctx context.Context, rfs renderFS, scratchDir string, relPaths []model.String, v walkAndModifyVisitor) error {
+//
+// rawPaths is expected to have been cleaned already from processPaths.
```

Comment:
Maybe update the docs for walkAndModify to mention globs

Reply:
Updated to mention processGlobs.